src/gmt_remote.c

```diff
@@ -72,9 +72,11 @@ GMT_LOCAL int give_data_attribution (struct GMT_CTRL *GMT, const char *file) {
 	/* Print attribution when the @earth_relief_xxx.grd file is downloaded for the first time */
 	char tag[4] = {""};
 	int k, match = -1, len = (int)strlen(file);
-	strncpy (tag, &file[len-3], 3U);
+	if (strstr (file, ".grd")) len -= 4;		/* If .grd extension was provided we must skip that as well */
+	else if (strstr (file, ".nc")) len -= 3;	/* If .nc extension was provided we must skip that as well */
```

Comment:
Does this mean I can use `gmt which -Gu @earth_relief_20m.nc`?

It gives some errors:
```
$ gmt which -Gu @earth_relief_20m.nc
gmtwhich [ERROR]: Libcurl Error: HTTP response code said error
gmtwhich [ERROR]: File earth_relief_20m.nc.grd not found!
```

Reply:
No, I was being tired and stupid.  I just removed that.  The files are *.grd but users should only specify @earth_relief_xxy with no extension.